README.md

```diff
@@ -27,32 +27,46 @@ Client client = new Client(0, new String[] { "127.0.0.1:3001", "127.0.0.1:3002",
 
 ### Account Creation
 
-All TigerBeetle's ID are 128-bit integer, and `tigerbeetle-java` uses a `java.util.UUID` to represent this value.
+All TigerBeetle's IDs are 128-bit unsigned integers that can be represented as a `byte[]` array of 16 bytes, a pair of `long` or a `java.util.UUID`.
 
 ```java
 import com.tigerbeetle.Client;
-import com.tigerbeetle.Account;
-import com.tigerbeetle.AccountsBatch;
-import com.tigerbeetle.CreateAccountsResult;
+import com.tigerbeetle.Accounts;
+import com.tigerbeetle.CreateAccountResults;
 
 // ...
 
 // Creates a batch with the desired capacity
-AccountsBatch batch = new AccountsBatch(100);
+Accounts batch = new Accounts(100);
```

Comment:
Calling this AccountsBatch is clearer IMO

Reply:
It's hard to name things! I choose to remove the "Batch" suffix from everything that inherits from Batch, I thought keeping it would be a bit noisy:

```Java
new AccountsBatch(...)
new TransfersBatch(...)
new CreateTransfer ResultsBatch(...)
new CreateAccountsResultsBatch(...)
new IdsBatch(...)
```

What do you think?

Reply:
haha good point! I think it's easier to _read_ but yes it's more verbose to write.

I might suggest though if you do have everything with *Batch is that you use the singular before. So `AccountBatch`, `TransferBatch`, `IdBatch`.